### PR TITLE
Update README.md

### LINKED ISSUE #105: Add relevant badges to README
https://shields.io/

### PR DESCRIPTION
**Pull request by mfix22**
- Closes #105

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,10 @@ You know [all](https://twitter.com/dan_abramov/status/890191815567175680) [of](h
 
 Visit [carbon.now.sh](https://carbon.now.sh) or read [our post](https://dawnlabs.io/blog/carbon/) to learn more about the project.
 
+[![styled with prettier](https://img.shields.io/badge/styled_with-prettier-ff69b4.svg)](https://github.com/prettier/prettier)
+[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](http://makeapullrequest.com)
+[![MIT License](https://img.shields.io/npm/l/kcd-scripts.svg)](https://github.com/dawnlabs/carbon/blob/master/LICENSE)
```
Comment (andrewda) [inline README.md]: Looks like the wrong repo here

Comment (andrewda) [reply README.md]: https://img.shields.io/github/license/dawnlabs/carbon.svg

**Comment by jakedex** [comment]
@yeskunall I think we should keep the features section, but I'm with you in that the usage section does feel a bit repetitive. Maybe we can work on cleaning that up? 

@mfix22 Could you add a badge for travis? Maybe [this](https://david-dm.org/) as well?

**Comment by yeskunall** [comment]
@jakedex sure thing. I'll look into it, but this PR might get merged before that, in which case I'll make a separate PR cleaning up that section of the README.

Cheers! ❤️

**Pull request merged**